### ISSUE TITLE
Makes possible to thread the config of the vite build via the plugin Options

### ISSUE THREAD

**Issue by sledorze** [sevenwestmedia-labs/nx-plugins#57]
I'm open to do a PR if it would be accepted

**Comment by JakeGinnivan** [comment]
@sledorze yep, would definitely accept an improvement around this

**Comment by sledorze** [comment]
@JakeGinnivan the PR provides a quick insight of what I meant - would that be in line with your thinking?

**Comment by sledorze** [comment]
Oh.. yes, this was my original issue; having a typo `vit.config.js` instead of `vite.config.js`
https://github.com/sevenwestmedia-labs/nx-plugins/pull/60/commits/2dec854553f75f549eaa438e55a5d9ce34c2d325

We can probably not set the config option if not provided then..
I have no more use case; do you still want we do it?

**Comment by JakeGinnivan** [comment]
Yeah. I think adding the option is good. If you could just make it optional that would be awesome @sledorze
